ccp/__init__.py

```diff
@@ -34,32 +35,49 @@
 
 HERE = os.path.abspath(os.path.dirname(__file__))
 
+Framework = namedtuple('Framework', ['name', 'path'])
 
 def main():
     sanity_check()
 
-    parser = argparse.ArgumentParser(description="Exclude dependencies")
+    parser = argparse.ArgumentParser(description="Argument Parser")
     parser.add_argument(
         "-x", "--exclude", nargs="+",
         help="Exclude dependencies from being copied", required=False)
 
+    parser.add_argument(
+        "-c", "--carthage", nargs="+",
+        help="Additional Carthage Binary Folders", required=False)
+
     args = parser.parse_args()
 
     if args.exclude is not None:
         excluded_frameworks = args.exclude
     else:
         excluded_frameworks = []
 
+    if args.carthage is not None:
+        additional_carthage_folders = args.carthage
+    else:
+        additional_carthage_folders = []
+
     built_products_dir = os.environ["BUILT_PRODUCTS_DIR"]
     frameworks_folder_path = os.environ["FRAMEWORKS_FOLDER_PATH"]
     srcroot = os.environ["SRCROOT"]
 
     dest = os.path.join(built_products_dir, frameworks_folder_path)
     frameworks_dir = os.path.abspath(os.path.join(srcroot, "Carthage", "Build", "iOS"))
```

Comment:
This needs to be in the else condition of `args.carthage is not None:`, otherwise you may get invalid path if the default carthage path is not valid